packages/conf/src/index.ts

```diff
@@ -241,22 +276,218 @@ export function replaceRootDir(
 }
 
 /**
- * Replace fallbackLanguage with fallbackLocale
- *
- * Released in lingui-conf 0.9
- * Remove anytime after 3.x
+ * Replace fallbackLanguage and fallbackLocale, by the new standard fallbackLocales
+ * - https://github.com/lingui/js-lingui/issues/791
+ * - Remove anytime after 3.x
```

Comment:
```suggestion
 * - Remove anytime after 4.x
```

We should bump the version here. The idea is that these deprecations shouldn't break current version, but they should be included in 4.x release notes. After that we can remove them.